Changes to Colab Notebook for Python 3.10

basebala opened the pull request:
Installations required to run the Colab notebook with Python 3.10. Reinstalls/reloads colabx-term, changes environment Python version, and removes python3-blinker. (Allows for training with preset data (processing custom images/video still doesn't work due to separate COLMAP issue).

Commit:
all colab changes

akristoffersen reviewed (approved):
This makes sense to me! Is there a reason we want to move down to numpy 1.x? I believe nerfstudio works fine with 2.0.

basebala commented:
> This makes sense to me! Is there a reason we want to move down to numpy 1.x? I believe nerfstudio works fine with 2.0.

Oh, perhaps it works now in colab. I think the issue was colab was not working with it earlier when I was initially working with the notebook which was almost 2 months ago, and that issue is documented here: https://github.com/nerfstudio-project/nerfstudio/issues/3288.

basebala committed:
Merge branch 'main' into colab_changes

basebala committed:
Remove forced reinstall of numpy

Removed forced reinstall of numpy version less than 2.0.0.

basebala committed:
Fix pip install command for numpy version

basebala committed:
Change error from array to float in test cases

basebala committed:
Upgrade pip and install 'wheel' in Dockerfile

Added installation of 'wheel' package and upgraded 'setuptools' in Dockerfile.

basebala committed:
Upgrade pip and install additional dependencies

Updated pip installation commands to include 'packaging' and fixed formatting issues.

basebala committed:
Refactor pip installation commands in Dockerfile

Updated pip installation commands and dependencies in Dockerfile.

basebala committed:
Add setuptools upgrade and additional pip installs

Updated Dockerfile to install additional Python packages and upgrade setuptools.

basebala committed:
Update pip install commands in Dockerfile

Force reinstall torch and torchvision with specific versions.

basebala committed:
Update pip install command for gsplat

basebala committed:
Optimize Dockerfile by removing duplicate pip install

Removed redundant pip install command for torch and torchvision.
